here-naksha-lib-base/src/commonMain/kotlin/naksha/base/Platform.kt

```diff
@@ -458,6 +458,8 @@ expect class Platform {
          * @return The inflated (decompress) bytes.
          */
         fun gzipInflate(compressed: ByteArray, bufferSize: Int = 0, offset: Int = 0, size: Int = Int.MAX_VALUE): ByteArray
+
+        var parseDataUrl: Boolean
```

Comment:
No way, this is a thread safe singleton, we do not add configurations into here, except really global. This is something that changes on every single invocation, please remove it from here!